MapleServer2/PacketHandlers/Game/RequestUserEnvHandler.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Packets;
+using MapleServer2.Servers.Game;
+using Microsoft.Extensions.Logging;
+
+namespace MapleServer2.PacketHandlers.Game
+{
+    public class RequestUserEnvHandler : GamePacketHandler
+    {
+        public override RecvOp OpCode => RecvOp.REQUEST_USER_ENV;
+
+        public RequestUserEnvHandler(ILogger<RequestUserEnvHandler> logger) : base(logger) { }
+
+        private enum UserEnvMode : byte
+        {
+            Unk = 0x0,
```

Comment:
Same here, this can be removed if we don't have a name for it and it's not being handled